debian/ubuntu-advantage-tools.postinst

```diff
@@ -60,7 +60,9 @@ redact_ubuntu_release_from_ua_apt_filenames() {
     DIR=$1
     UA_SERVICES=$(/usr/bin/python3 -c "
 from uaclient.entitlements import valid_services
-print(*valid_services(allow_beta=True, all_names=True), sep=' ')
+from uaclient.config import UAConfig
+cfg = UAConfig()
+print(*valid_services(cfg=cfg, allow_beta=True, all_names=True), sep=' ')
 ")
```

Comment:
This `redact_ubuntu_release_from_ua_apt_filenames` function is only for an old migration of apt filenames. Can we just hardcode this list to the current list of services? (new services will never need to undergo this migration)

Reply:
And sorry, that's not really relevant to this PR, but since we're touching this anyway, it might be a good time :) (but maybe it deserves it's own PR?)

Reply:
I completely agree we can change this, but I think we could do this in a separate PR